test/test.py

```diff
@@ -0,0 +1,11 @@
+"""
+CVE-bin-tool util tests
+"""
+import unittest
+
+from cve_bin_tool.util import inpath
+
+class TestUtil(unittest.TestCase):
+
+    def test_inpath(self):
```

Comment:
You should re-name this to `test_not_inpath`, move it to `test/test_util.py` and change 'python' to some random string